scripts/integration-tests/verdaccio-config.yml

```diff
@@ -20,6 +20,10 @@ auth:
 uplinks:
   npmjs:
     url: https://registry.npmjs.org/
+    agent_options:
```

Comment:
The `agent_options` are not need it if you use (v4.10.0). It is harmless if you like to keep it.